tests/test_summary.py

```diff
@@ -0,0 +1,91 @@
+with patch('webviz_config.common_cache.cache.memoize',
+           lambda *x, **y: lambda f: f):
+    from webviz_subsurface.datainput import \
+        load_ensemble_set, \
+        get_time_series_statistics, \
+        get_time_series_fielgains, \
+        get_time_series_data
+
+
+# define recurring variables
+ENSEMBLE_PATHS = [
+    ('iter--0', '/path/to/ensemble/realization-*/iter-0'),
+    ('iter--1', '/path/to/ensemble/realization-*/iter-1'),
+    ('iter--2', '/path/to/ensemble/realization-*/iter-2'),
+]
+ENSEMBLESET_NAME = 'Test Ensemble'
+TIME_INDEX = 'yearly'
+COLUMN_KEYS = ['FOP*', 'FGP*']
+BASE_ENSEMBLES = ['iter--0']
+DELTA_ENSEMBLES = ['iter--0', 'iter--1', 'iter--2']
+
+
+@pytest.mark.skipif('fmu.ensemble' not in sys.modules,
+                    reason="Requires fmu.ensemble installed")
+def test_load_ensemble_set():
+
+    ensset = load_ensemble_set(
+        ensemble_paths=ENSEMBLE_PATHS,
+        ensemble_set_name=ENSEMBLESET_NAME
+    )
+    assert len(ensset) == 3
+    assert len(ensset["iter--0"].get_df("STATUS")) == 125
+    assert len(ensset["iter--1"].get_df("STATUS")) == 105
+    assert len(ensset["iter--2"].get_df("STATUS")) == 85
+
+
+@pytest.mark.skipif('fmu.ensemble' not in sys.modules,
+                    reason="Requires fmu.ensemble installed")
+def test_time_series_statistics():
+
+    summary_statistics = get_time_series_statistics(
+        ensemble_paths=ENSEMBLE_PATHS,
+        time_index=TIME_INDEX,
+        column_keys=COLUMN_KEYS
+    )
+    assert isinstance(summary_statistics, pd.DataFrame)
+    assert summary_statistics.shape == (165, 5)
+
+
+@pytest.mark.skipif('fmu.ensemble' not in sys.modules,
+                    reason="Requires fmu.ensemble installed")
+def test_time_series_data():
+
+    summary_data = get_time_series_data(
+        ensemble_paths=ENSEMBLE_PATHS,
+        time_index=TIME_INDEX,
+        column_keys=COLUMN_KEYS,
+        ensemble_set_name=ENSEMBLESET_NAME
+    )
+    assert isinstance(summary_data, pd.DataFrame)
+    assert summary_data.shape == (693, 7)
+
+
+@pytest.mark.skipif('fmu.ensemble' not in sys.modules,
+                    reason="Requires fmu.ensemble installed")
+def test_get_time_series_fielgains():
+
+    field_gains = get_time_series_fielgains(
+        ensemble_paths=ENSEMBLE_PATHS,
+        time_index=TIME_INDEX,
+        column_keys=COLUMN_KEYS,
+        base_ensembles=BASE_ENSEMBLES,
+        delta_ensembles=DELTA_ENSEMBLES,
+        ensemble_set_name=ENSEMBLESET_NAME
+    )
+    assert isinstance(field_gains, pd.DataFrame)
+    assert field_gains.shape == (825, 7)
+    assert all([column in field_gains.columns
+                for column in ['IROENS - REFENS', 'REAL', 'DATE']])
+
+    iorens, refens = 'iter--0', 'iter--1'
+    compared_ensembles = f'{iorens} - {refens}'
+    field_gain = field_gains[
+        field_gains['IROENS - REFENS'] == compared_ensembles]
+    assert isinstance(field_gain, pd.DataFrame)
+    assert field_gain.shape == (275, 7)
```

Comment:
This file I guess is assuming that you have a very specific test data set at hand (since the tests basically checks that the input/output has some preknown shapes)? Maybe we can for now delete this file (it will be in the commit history) + most of it should anyway by covered by tests in `fmu-ensemble` perhaps?

Reply:
Agree. Is there a way to check if api-calls are performed correct?